libs/components/tabs/src/lib/modules/sectioned-form/sectioned-form.component.ts

```diff
@@ -64,62 +64,68 @@ export class SkySectionedFormComponent
   @Output()
   public indexChanged: EventEmitter<number> = new EventEmitter();
 
-  public get ariaRole(): string {
-    return this.isMobile ? undefined : 'tablist';
-  }
+  public ariaRole: string | undefined = 'tablist';
 
   @ViewChild('skySectionSideContent')
-  public content: ElementRef;
+  public content: ElementRef | undefined;
+
+  #ngUnsubscribe = new Subject<void>();
 
-  private isMobile = false;
-  private _ngUnsubscribe = new Subject<void>();
+  #changeRef: ChangeDetectorRef;
 
   constructor(
     public tabService: SkyVerticalTabsetService,
-    private changeRef: ChangeDetectorRef
-  ) {}
+    changeRef: ChangeDetectorRef
+  ) {
+    this.#changeRef = changeRef;
+  }
 
-  public ngOnInit() {
+  public ngOnInit(): void {
     this.tabService.maintainTabContent = this.maintainSectionContent;
 
     this.tabService.indexChanged
-      .pipe(takeUntil(this._ngUnsubscribe))
+      .pipe(takeUntil(this.#ngUnsubscribe))
       .subscribe((index) => {
         this.indexChanged.emit(index);
-        this.changeRef.markForCheck();
+        this.#changeRef.markForCheck();
       });
 
     this.tabService.switchingMobile
-      .pipe(takeUntil(this._ngUnsubscribe))
+      .pipe(takeUntil(this.#ngUnsubscribe))
       .subscribe((mobile: boolean) => {
-        this.isMobile = mobile;
-        this.changeRef.markForCheck();
+        if (!mobile) {
+          this.ariaRole = 'tablist';
+        } else {
+          this.ariaRole = undefined;
+        }
+
+        this.#changeRef.markForCheck();
       });
 
     if (this.tabService.isMobile()) {
-      this.isMobile = true;
+      this.ariaRole = undefined;
       this.tabService.animationContentVisibleState = VISIBLE_STATE;
-      this.changeRef.markForCheck();
+      this.#changeRef.markForCheck();
```

Comment:
Can this logic not be brought into the subscription for switchingMobile?

Reply:
No. We need to do both - we watch for changes to the mobile state in the `switchingMobile` subscription but also have to react to the initial state. It could be reworked by using a `ReplaySubject` in the backing service; however, that would be outside the scope of this work.